docs/adding-packages.md

```diff
@@ -64,7 +64,7 @@ The attributes supported by `info` are:
 
 All of these are optional.
 
-## <span id="user-content-using-info">Using `info` as a path</span>
+## [Using `info` as a path]{#user-content-using-info}
```

Comment:
More on this syntax,
https://github.com/jgm/commonmark-hs/blob/master/commonmark-extensions/test/attributes.md

Emanote supports most of these markdown extensions,
https://github.com/jgm/commonmark-hs/tree/master/commonmark-extensions

Reply:
How do you do an admonition?

Reply:
Admonitions are currently simulated using attributes extension: https://emanote.srid.ca/guide/markdown/custom-style

But this is probably the proper way to do it: https://github.com/EmaApps/emanote/discussions/307